### PR TITLE
LLM: fix outlier value

### PR DESCRIPTION
**Pull request by WeiguangHan**
@hkvision Fix outlier value such as rest token latency is 0.0.

### CODE REVIEW

File: python/llm/test/benchmark/csv_to_html.py [hunk 1]
```diff
@@ -30,6 +30,10 @@ def highlight_vals(val, max=3.0, color1='red', color2='green'):
     else:
         return ''
 
+def first_nonzero_min(lst):
```
Comment (hkvision) [inline python/llm/test/benchmark/csv_to_html.py:33]: just nonzero_min?

File: python/llm/test/benchmark/csv_to_html.py [hunk 2]
```diff
@@ -30,6 +30,10 @@ def highlight_vals(val, max=3.0, color1='red', color2='green'):
...
+    non_zero_lst = [num for num in lst if num != 0.0]
```
Comment (hkvision) [inline python/llm/test/benchmark/csv_to_html.py:34]: if num <= 0.0?

File: python/llm/test/benchmark/csv_to_html.py [hunk 3]
```diff
@@ -40,8 +44,8 @@ def add_to_dict(dict, key, value):
 
 def best_in_dict(dict, key, value):
     if key in dict:
-        best_value=min(dict[key])
-        if best_value < value:
+        best_value = first_nonzero_min(dict[key])
+        if best_value < value or value == 0.0:
```
Comment (hkvision) [inline python/llm/test/benchmark/csv_to_html.py:48]: value <=0.0?

File: python/llm/test/benchmark/csv_to_html.py (hunk 1 above)
Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py]: ok

File: python/llm/test/benchmark/csv_to_html.py (hunk 2 above)
Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py]: ok

File: python/llm/test/benchmark/csv_to_html.py (hunk 3 above)
Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py]: ok

**Review by hkvision — APPROVED** [review]

**Pull request merged**